contracts/registry/JurorsRegistry.sol

```diff
@@ -955,7 +987,7 @@ contract JurorsRegistry is ControlledRecoverable, IJurorsRegistry, ERC900, Appro
     */
     function _jurorBrightId(address _jurorSenderAddress) internal view returns (address) {
         if (_jurorSenderAddress == ZERO_ADDRESS) {
-            return ZERO_ADDRESS;
+            return ZERO_ADDRESS; // TODO: Can this be removed.
```

Comment:
I think we can remove it yes, the only possible entry point with `_jurorSenderAddress ` being `address(0)` is through `_updateBrightIdActiveStake` and i  can't see a case where it wouldn't revert before getting to this function.

Reply:
Will also leave for now because I know it will make some tests fail. Will investigate further once it's finished.